lib/puppet_x/templates/init_ps.ps1.erb

```diff
@@ -476,7 +486,14 @@ function Invoke-PowerShellUserCode
       # is to return 1 so that we ensure Puppet reports this run as an error.
       $ec = 1
     }
-    $output = $_.Exception.Message | Out-String
+
+    if ($_.Exception.ErrorRecord.InvocationInfo -ne $null)
+    {
+      $output = $_.Exception.Message + "`n`r" + $_.Exception.ErrorRecord.InvocationInfo.PositionMessage
```

Comment:
Verified it works as expected on Server 2008

Error in raw powershell 
``` powershell
Windows PowerShell
Copyright (C) 2009 Microsoft Corporation. All rights reserved.

PS C:\Users\Administrator> & { $ErrorActionPreference = 'Stop'; $test = 1/0 }
Attempted to divide by zero.
At line:1 char:48
+ & { $ErrorActionPreference = 'Stop'; $test = 1/ <<<< 0 }
    + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : RuntimeException

PS C:\Users\Administrator> $PSVersionTable

Name                           Value
----                           -----
CLRVersion                     2.0.50727.4253
BuildVersion                   6.0.6002.18111
PSVersion                      2.0
WSManStackVersion              2.0
PSCompatibleVersions           {1.0, 2.0}
SerializationVersion           1.1.0.1
PSRemotingProtocolVersion      2.1


PS C:\Users\Administrator>
```

Same code but in Puppet debug
```
Debug: 2016-09-01 14:46:10 -0700 PIPE>

Debug: 2016-09-01 14:46:10 -0700 PIPE>

Debug: Wait object signaled
Debug: Waited 100 total milliseconds.
Debug: STDERR: Attempted to divide by zero.

At line:1 char:44
+ $ErrorActionPreference = "Stop"; $test = 1/ <<<< 0
Error: $ErrorActionPreference = "Stop"; $test = 1/0 returned 1 instead of one of [0]
Error: /Stage[main]/Main/Exec[cwd test]/returns: change from notrun to 0 failed: $ErrorActionPreference = "Stop"; $test = 1/0 returned 1 instead of one of [0]
Debug: Class[Main]: Resource is being skipped, unscheduling all events
Debug: Stage[main]: Resource is being skipped, unscheduling all events
Debug: Finishing transaction 55567200
Debug: Storing state
```